WordPress/gutenberg
# Experiment: Replicate Navigation block using directives (Alpinejs)

## What?
This is an experiment to replicate the functionalities of the navigations block but using directives. In this case we are using Alpinejs. This is part of the [directives hydration experiment](https://github.com/WordPress/block-hydration-experiments/issues/60).

## Why?
In the [directives hydration experiment](https://github.com/WordPress/block-hydration-experiments/issues/60) we have started using directives, and we want to understand which ones would be needed for this use case.

It is worth mentioning that this is just an experiment with learning purposes. We want to understand better the directives that are needed and I am not suggesting we should use `AlpineJS` instead of `micromodal`.

## How?
I removed the `micromodal` library and added AlpineJS (with the `@alpine/focus` plugin) instead. From there, I am just adding the directives needed to replicate the behaviour.

## Testing Instructions
Just open a site with the navigation block and check everything is working as expected.
